orderer/consensus/etcdraft/chain.go

```diff
@@ -1139,9 +1139,12 @@ func (c *Chain) apply(ents []raftpb.Entry) {
 		}
 	}
 
-	if c.accDataSize >= c.sizeLimit {
-		b := protoutil.UnmarshalBlockOrPanic(ents[position].Data)
+	// at postion==0, ents[position].Type is ambiguous, it can be either of {raftpb.EntryNormal, raftpb.EntryConfChange}
+	// take a snapshot only for ents[position].Type == raftpb.EntryNormal
+	isNormalEntry := ents[position].Type == raftpb.EntryNormal
```

Comment:
can we combine this statement into one if condition.  what is the intention of keeping it 2 statements?
> if c.accDataSize >= c.sizeLimit && ents[position].Type == raftpb.EntryNormal && len(ents[position].Data) > 0 {